skietbaan_fe/src/components/ViewComp.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
-import { fetchcomp, updateByIDcomp } from '../actions/competition.action';
+import { fetchcomp, 
```

Comment:
Fix the indentation
```
import {
 fetchcomp,
 updateByIDcomp 
} from '../actions/competition.action';
```

Reply:
Thought, I double checked. Will resolve

Reply:
look at my import again please you have not done as requested